index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kip's portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header-container">
+        <div class="header-img-left">
+            <img src="images/image_geometry_1.png" alt="image">
+        </div>
+        <nav class="header-nav-lg">
+            <img src="images/ic_menu.jpg" alt="image">
+        </nav>>
```

Comment:
- [ ] I couldn't see your hamburger button on the top right of your page. Please fix it. :writing_hand: 
![1](https://user-images.githubusercontent.com/70483835/154419223-844c80e5-9c23-440e-a5cb-e5795495e712.png)